_packages/@karrotmarket/gatsby-theme-global-website/package.json

```
@@ -23,6 +23,8 @@
     "@karrotmarket/gatsby-theme-website": "workspace:^",
     "@karrotmarket/react-stitches": "0.1.1-1",
     "@reach/router": "1.3.4",
+    "@seed-design/design-token": "^1.0.0-alpha.0",
+    "@seed-design/stylesheet": "^1.0.0-alpha.0",
```

버전이 달라요! alpha 채널로 받는게 좋을 거 같아요

```suggestion
    "@seed-design/design-token": "alpha",
    "@seed-design/stylesheet": "alpha",
```

---

아하 변경해놓을게요~!